### PR TITLE
Added more display options

### PR DESCRIPTION
**Pull request by df-gss**
### What was happening

- Requirement for more build-in display options

### What was done
- Added more display options to scss

### Checklist

-   [x] tested locally

### CODE REVIEW

**Comment by gnbm** [comment]
Hello [@df-gss ](https://github.com/df-gss):wave:
Thanks for your Pull Request. It looks like this may be your first contribution to an OutSystems Open Source project. Before we look into your contribution, we need you to accept the :memo: [OutSystems Contributors License Agreement](https://outsystems-public-dl.s3-eu-west-1.amazonaws.com/opensource/cla/OutSystems-Contributor-License-Agreement-2020-03-23.pdf).
After you read and accept the terms of the agreement, reply to this message saying `I read, understood and accept the OutSystems CLA`.

After this step, we'll review your Pull Request and provide any feedback if needed. 
Thank you and feel free to keep contributing! 🚀

**Comment by df-gss** [comment]
Hi @gnbm , I have contributed before, this is not my first time.

#832 
#716

**Comment by gnbm** [comment]
> Hi @gnbm , I have contributed before, this is not my first time.
> 
> #832 #716

Hi @df-gss 👋🏻
I remember that and thank you for another contribution.
You can ignore the other automatic message.
Cheers

**Review by gnbm — APPROVED** [review]
LGTM 👌🏻

**Review by rugoncalves — APPROVED** [review]

File: src/scss/05-useful/_display.scss [hunk 1]
```diff
@@ -5,11 +5,31 @@
...
+
+	&-inline-block {
+		display: inline-block;
+	}
+
+	&-inline-flex {
+		display: inline-flex;
+	}
+
+	&-inline-grid {
```
Comment (BenOsodrac) [inline src/scss/05-useful/_display.scss:32]: I would call this one just `&-grid` and remove the inline

**Review by BenOsodrac — CHANGES_REQUESTED** [review]

File: src/scss/05-useful/_display.scss (hunk 1 above)
Comment (gnbm) [reply src/scss/05-useful/_display.scss:32]: @df-gss can you please review this before we merge the code? 
Thank you

Comment (df-gss) [reply src/scss/05-useful/_display.scss]: I changed it. It should indeed be called 'display-grid'.

**Review by gnbm — APPROVED** [review]

**Review by BenOsodrac — APPROVED** [review]

**Comment by gnbm** [comment]
@df-gss To be able to merge PRs, all the commits are required to be signed for compliance reasons.

**Comment by df-gss** [comment]
> @df-gss To be able to merge PRs, all the commits are required to be signed for compliance reasons.

Commit should be signed now.

**Comment by gnbm** [comment]
Hello [@df-gss ](https://github.com/df-gss)👋
Thank you for your contribution and we are glad to inform the PR was merged after our validation.
We will also update the Cheatsheet on our website after the next release which will include these new CSS utility classes.
Cheers!

**Comment by df-gss** [comment]
Hi, Any idea when the next release will be?

**Comment by gnbm** [comment]
> Hi, Any idea when the next release will be?
The ETA we have atm is April 1st

**Pull request merged**